src/pages/companion/profile-temp/[externalUserId]/index.tsx

```diff
@@ -0,0 +1,199 @@
+import { use, useEffect, useState } from "react";
+import Profile from '../../../../components/companion/blueprintProfiles/profile' // update these once done testing
+import ExtraInfo from "../../../../components/companion/blueprintProfiles/extraInfo";
+import { useRouter } from "next/router";
+import CompanyInfo from "../../../../components/companion/blueprintProfiles/companyInfo";
+import AttendeeInfo from "../../../../components/companion/blueprintProfiles/attendeeInfo";
+import { TopNav } from '../../../../components/companion/navigation/top-nav';
+import { SideNav } from '../../../../components/companion/navigation/side-nav';
+import { BottomNav } from '../../../../components/companion/navigation/bottom-nav';
```

Comment:
Izzy recently worked on a refactor for the navigation bars, so you might encounter merge conflicts here. No need to change yet but just a heads up

Reply:
@m20arcusk Could we adjust the Nav bars now that Izzy's changes are merged into `dev`? Will need to pull the latest `dev` changes and use the `NavBarContainer` component